mappings/net/minecraft/util/math/Direction.mapping

```diff
@@ -88,6 +88,8 @@ CLASS net/minecraft/class_2350 net/minecraft/util/math/Direction
 		ARG 0 matrix
 		ARG 1 direction
 	METHOD method_23955 getUnitVector ()Lnet/minecraft/class_1160;
+	METHOD method_30928 pointsTo (F)Z
```

Comment:
This rather returns if the current direction forms an acute angle with the given yaw value. Better note that this always return `false` for vertical directions.